src/dswx_sar/region_growing.py

```diff
@@ -54,15 +62,16 @@ def region_growing(likelihood_image,
     if mode == 'descending':
         if initial_threshold <= relaxed_threshold:
             err_str = f"Initial threshold {initial_threshold} " \
-                      f" should be larger than relaxed threshold" \
-                      f"{relaxed_threshold}."
+                        f" should be larger than relaxed threshold" \
+                        f"{relaxed_threshold}."
             raise ValueError(err_str)
-    else:
+    elif mode == 'acending':
```

Comment:
```suggestion
    elif mode == 'ascending':
```
Not sure if it's typo or not. Can you take a look?